.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Our policy is latest minus one, so let's update this to 3.11 and 3.12 only.

Reply:
updated